src/LinuxTracingIntegrationTests/IntegrationTestChildProcess.h

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef LINUX_TRACING_INTEGRATION_TESTS_INTEGRATION_TEST_CHILD_PROCESS_H_
+#define LINUX_TRACING_INTEGRATION_TESTS_INTEGRATION_TEST_CHILD_PROCESS_H_
+
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <array>
+#include <functional>
+#include <string>
+
+#include "OrbitBase/Logging.h"
+
+namespace orbit_linux_tracing_integration_tests {
+
+class ChildProcess {
```

Comment:
I guess we can take the opportunity to add a brief commend.

Reply:
+1

Reply:
Fair, done.